### PR TITLE
Update test_e2e.sh

### LINKED ISSUE #784: Ant-XGBoost end-to-end test prints too much log
It is over 100 lines.

```
$ docker run --rm --net=host -v /var/run/docker.sock:/var/run/docker.sock -v $HOME/.kube:/root/.kube -v /home/$USER/.minikube/:/home/$USER/.minikube/ -v $GOPATH:/go -w /go/src/github.com/sql-machine-learning/sqlflow sqlflow:latest bash scripts/test_e2e.sh
 * Starting MySQL database server mysqld
   ...done.
mysql: [Warning] Using a password on the command line interface can be insecure.
mysql: [Warning] Using a password on the command line interface can be insecure.
mysql: [Warning] Using a password on the command line interface can be insecure.
mysql: [Warning] Using a password on the command line interface can be insecure.
mysql: [Warning] Using a password on the command line interface can be insecure.
2019/09/05 06:28:43 Launch server with insecure mode.
2019/09/05 06:28:43 Server Started at :50051
[TerminalIPythonApp] WARNING | Unknown error in handling startup files:
1 row affected
0 row affected
2 rows affected
UsageError: Line magic function `%autoreload` not found.
.
----------------------------------------------------------------------
Ran 1 test in 0.039s
OK
[TerminalIPythonApp] WARNING | Unknown error in handling startup files:
time="2019-09-05T06:28:48Z" level=info msg="tensorflowCmd: run locally" package=sql
WARNING:root:======xgboost training config======
{
  "xgboost_conf": {
    "params": {
      "objective": "multi:softprob",
      "booster": "gbtree",
      "seed": 0,
      "num_class": 3,
      "eta": 0.3,
      "max_depth": 5,
      "tree_method": "approx",
      "nthread": 2,
      "convergence_criteria": "-1:0:1.0"
    },
    "num_boost_round": 30,
    "auto_train": false
  },
  "data_conf": {
    "data_source": {
      "name": "sqlflow",
      "config": {
        "is_train": true,
        "standard_select": "SELECT * FROM sf_home.sqlflow_training_train",
```

### PR DESCRIPTION
**Pull request by wangkuiyi**
Fix https://github.com/sql-machine-learning/sqlflow/issues/784

I would prefer to see that Ant-XGBoost either merges to dmlc/xgboost, or the launcher in Ant-XGBoost being separated out into alipay/automl.  It doens't make sense to me to keep a launcher of xgboost in a fork and release the fork under the name [`xgboost-launcher`](https://pypi.org/project/xgboost-launcher/).

Before the code clean up, I think we shouldn't keep the end-to-end test of Ant-XGBoost in the Travis CI of SQLFlow.

### CODE REVIEW

**Comment by Yancey0623** [comment]
Please skip the ant-xgbosot uni test in [executor_test.go](https://github.com/sql-machine-learning/sqlflow/blob/develop/sql/executor_test.go#L74) to pass the CI.

File: scripts/image_build.sh [hunk 1]
```diff
@@ -114,8 +114,9 @@ echo 'get_ipython().magic(u"%reload_ext sqlflow.magic")' >> $IPYTHON_STARTUP/00-
 echo 'get_ipython().magic(u"%autoreload 2")' >> $IPYTHON_STARTUP/00-first.py
 curl https://raw.githubusercontent.com/sql-machine-learning/sqlflow/develop/example/jupyter/example.ipynb --output /workspace/example.ipynb
 
-# 9. install xgboost-launcher
-pip install xgboost-launcher==0.0.4
+# Re-enable this after Ant-XGBoost is ready.
+# # 9. install xgboost-launcher
+# pip install xgboost-launcher==0.0.4
```
Comment (Yancey0623) [inline scripts/image_build.sh:120]: The `xgboost-laucnher` included the `dmlc/xgboost` package, if we want to skip ant-xgboost unit test, maybe we also need to install `xgboost==0.90` to pass the `dmlc/xgboost` unit test.

Comment (wangkuiyi) [reply scripts/image_build.sh:120]: Added. Thanks for the reminder!

**Comment by tonyyang-svail** [comment]
Hi @Yancey1989, would you please file a PR changing the Docker image from `launcher` to `xgboost`?

**Comment by Yancey0623** [comment]
Hi @tonyyang-svail @wangkuiyi , I reopen this PR, and I'm so sorry that I closed this PR as my hands slipped 😂😂

**Comment by sperlingxx** [comment]
LGTM

**Comment by Yancey0623** [comment]
> Hi @Yancey1989, would you please file a PR changing the Docker image from launcher to xgboost?

@tonyyang-svail will do that today 😄😄

**Review by Yancey0623 — APPROVED** [review]
LGTM

**Pull request merged**